app/src/main/cpp/configure_nettle.sh

```diff
@@ -1,4 +1,4 @@
-TOOLCHAIN=$ANDROID_HOME/ndk/$1/toolchains/llvm/prebuilt/$(uname -s | tr '[:upper:]' '[:lower:]')-x86_64/bin
+TOOLCHAIN=$ANDROID_NDK_HOME/toolchains/llvm/prebuilt/$(uname -s | tr '[:upper:]' '[:lower:]')-x86_64/bin
```

Comment:
This may not point to the required version of ndk

Reply:
CI host have its owned ndk?

Reply:
`ANDROID_NDK_HOME` may point to a ndk that's newer or older than what's specified in build script